### ISSUE TITLE
Not complete, but this is what I have

### ISSUE THREAD

**Issue by akcuster** [akcuster/plantParentHelper#7]
@pawaitemadisoncollege I've been running into a lot of issues with IntelliJ that are hindering me from finishing the project. Tests will randomly stop passing, preventing me from deploying and checking on changes. Sometimes they will work again when restarting IntelliJ, sometimes they won't. I've always had it tell me that an entity doesn't exist when it very much does and there have been no changes. Again, sometimes restarting fixes it, but not always. I've been unable to find a fix for the issues and almost completely broke my code while trying to figure out what's wrong. Up until these issues started, I had the AWS cognito login and log out working and I had the user profile set up to display the names of all plants in their collection. I also had the ability for a user to add a plant working correctly. Because I can no longer get my application deployed, I don't have screenshots or a walkthrough video and I had not gotten around to getting it out on AWS before everything started breaking. I have gone through and made sure that all of my javadoc comments are up to date and tried to add in additional comments to explain what the code is doing, along with TODO comments to explain what I still would like to add. I understand and accept that I likely will not get a passing grade because of this. I feel confident that under different circumstances, I would have been able to find the solution to get things working again.

Despite the disappointing outcome, I learned a lot from this project, both through successes, like getting cognito to work and setting up my database and the relationships between entities, and through things I would do differently, like taking more time to plan out the project before even writing a line of code.

**Comment by pawaitemadisoncollege** [comment]
Hi Amber (@akcuster), that sounds like a very frustrating last few days of dev work... If you turn off the unit tests in the maven view, are you able deploy locally so that you could record a walkthrough video? I understand if you need a break from it right now, but would you be willing to try for a local deploy for your video in the next couple days? I'd really like to be able to see this in action for a few reasons - like the fact that it's a such a neat app idea and I'd like to see how far you've gotten with it, and I'd really like to be able to review this having as much info as possible and a demo really, really helps me with that...

**Comment by akcuster** [comment]
Sorry I did not get back to you right away, I kind of needed a day to just step away and get out of my apartment. I was able to get it deployed locally after disabling the tests. I'm still not sure what's happening with those, but all the functionality I have at this point seems to be working. I appreciate how flexible you've been with me on this. I'm going to try to clean it up a little and then I'll get a demo video to you by this evening.
